crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -256,10 +257,7 @@ impl ShardedGossip {
                             state.throughput.op_bloom_bytes.outgoing +=
                                 missing_hashes.size() as u32;
                         }
-                        ShardedGossipWire::OpRegions(OpRegions { region_set, .. }) => {
-                            state.throughput.region_bytes.outgoing +=
-                                region_set.regions().map(|r| r.data.size).sum::<u32>();
-                        }
+                        ShardedGossipWire::OpRegions(OpRegions { region_set: _, .. }) => {}
```

Comment:
```suggestion
                        ShardedGossipWire::OpRegions(OpRegions { region_set: _, .. }) => (),
```

Reply:
Is it correct that this is no longer doing anything?

Reply:
Yes, at first it was doing something, and then it wasn't, so I kept the doing-nothing explicit.